engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelHydrationValidation.kt

```diff
@@ -73,13 +78,36 @@ internal class NadelHydrationValidation(
         overallField: GraphQLFieldDefinition,
         actorService: Service,
         actorField: GraphQLFieldDefinition,
+        isPolymorphicHydration: Boolean,
```

Comment:
who sets this to be true?  The AGG?

Reply:
I can see it being used

Reply:
it's from here https://github.com/atlassian-labs/nadel/pull/367/files#diff-6530fb55bf6b4af9221e42e6ba6ba5070794d407cc4652bc7898f357e6ea1b6aR70